libs/blocks/card/card.css

```diff
@@ -2,3 +2,11 @@
 .card .consonant-ProductCard-title {
   line-height: 1.375rem;
 }
+
+hr {
```

Comment:
This should have more specification so it doesn't create collisions
```suggestion
.card hr {
```

Reply:
good point - thx!